src/components/Checkbox.jsx

```diff
@@ -10,13 +10,15 @@ const Checkbox = ({ toggle, onClick = () => {}, text = "", color }) => {
       data-cy="checkbox"
     >
       <div
+        data-cy="checkbox-blue"
```

Comment:
why is it blue?

Reply:
I thought I changed that, i'll change it to "bg"